### ISSUE TITLE
send dt printing to logger

### ISSUE THREAD

**Issue by jeanconn** [sot/xija#74]
Not really an issue, but I note that there's a lot of printing of the dt during the module tests:

```PASSED
xija/tests/test_models.py::test_dpa Using dt = 328 s.
PASSED
xija/tests/test_models.py::test_plotdate Using dt = 328 s.
PASSED
xija/tests/test_models.py::test_data_types Using dt = 328 s.
Using dt = 328 s.
Using dt = 328 s.
Using dt = 328 s.
Using dt = 328 s.
Using dt = 328 s.
Using dt = 328 s.
PASSED
xija/tests/test_models.py::test_minusz Using dt = 328 s.
PASSED
xija/tests/test_models.py::test_pftank2t Using dt = 328 s.
PASSED
xija/tests/test_models.py::test_multi_solar_heat_values Using dt = 328 s.
Using dt = 328 s.
PASSED
```

_Originally posted by @jeanconn in https://github.com/sot/xija/pull/48#issuecomment-578236181_

**Comment by jeanconn** [comment]
@matthewdahmer it sounds like if this is having an impact for you priority is higher.

**Comment by jzuhone** [comment]
@jeanconn this is fixed in PR #75.

**Comment by jeanconn** [comment]
Thanks @jzuhone.  I just wasn't sure when we were going to get a chance to review and run with #75.  It sounds like from @matthewdahmer 's notes that he can work around if this doesn't go this week.

**Comment by matthewdahmer** [comment]
Thank you I do have a workaround where I can use an older version of Xija to produce the data I need, however this bug otherwise would have a significant impact.

**Comment by jzuhone** [comment]
This issue can be closed now.

**Comment by jeanconn** [comment]
Closed by #75
